src/npc.cpp

```diff
@@ -104,6 +104,36 @@ npc::npc()
     last_updated = calendar::turn;
 }
 
+standard_npc::standard_npc( const std::string &name, const std::vector<itype_id> &clothing,
+                            int sk_lvl, int s_str, int s_dex, int s_int, int s_per )
+{
+    this->name = name;
```

Comment:
Why don't you use initialization lists for the simple assignments? `name( name )` looks more consistent (usage of `this->` semantics is much rearer in our code). This also prevents creating an empty string implicitly on construction (`name()`).